app/views/catalog/_index_masonry.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="masonry document col-6 col-md-3">
   <div class="thumbnail">
-    <%= render_thumbnail_tag(document, {}, :counter => document_counter_with_offset(document_counter)) %>
+    <%= render_thumbnail_tag(document, { class: 'img-thumbnail' }, counter: document_counter_with_offset(document_counter)) %>
```

Comment:
isn't `render_thumbnail_tag` deprecated in blacklight?

Reply:
🤷‍♂. All I care about in this PR is making sure it actually displays OK under bootstrap 4.